### DISCUSSION TITLE
I can't use tailwind in my laravel project

### DISCUSSION THREAD

**Discussion by Yasbarros** [tailwindlabs/tailwindcss#13571] [Help] [answered]
Laravel is not recognizing tailwind classes

**Comment by wongjn** [comment]
Consider checking you have followed [the installations instructions for Laravel](https://tailwindcss.com/docs/guides/laravel). In particular, consider ensuring you have added the correct `content` file globs for your project.

**Reply by Yasbarros** [reply, marked as answer]
Thank you very much for your feedback! I managed to solve it, I needed to install vite.config.js and put a server key with the localhost URL itself and install the postcss file with tailwind in the plugin
